spec/pq/authentication_methods_spec.cr

```diff
@@ -9,38 +9,65 @@ require "../spec_helper"
 
 describe PQ::Connection, "nologin role" do
   it "raises" do
-    DB.exec("drop role if exists crystal_test")
-    DB.exec("create role crystal_test nologin")
+    PG_DB.exec("drop role if exists crystal_test")
+    PG_DB.exec("create role crystal_test nologin")
     expect_raises(PQ::PQError) {
-      PG::Connection.new("host=localhost user=crystal_test")
+      DB.open("postgres://crystal_test@localhost")
     }
-    DB.exec("drop role if exists crystal_test")
+    PG_DB.exec("drop role if exists crystal_test")
   end
 end
 
 if File.exists?(File.join(File.dirname(__FILE__), "../.run_auth_specs"))
+  describe PQ::Connection, "cleartext auth" do
```

Comment:
did the cleartext auth get readded from your branch? I removed it previously to prevent a mitm attack, and also there aren't really any servers left that don't support the md5 auth.